crypto/default/pom.xml

```diff
@@ -52,7 +52,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <scope>provided</scope>
```

Comment:
The reason I made the change here and not under integration/{admin,client}-cli is that the reason these packages are needed are purely because they are used by the crypto module itself. I will move it to the cli pom, but where the deps between different crypto modules differs, we are pulling in unnecessary dependencies into the cli.  For example, the httpclient dep is not included in the elytron module.

Reply:
Ah, ok. Thanks for the explain. So maybe it is better to keep this one as is then?

Reply:
I went ahead and made the change you requested. Perhaps this can be revisited at a later date.